MoltenVK/MoltenVK/GPUObjects/MVKDeviceMemory.h

```diff
@@ -52,7 +52,7 @@ class MVKDeviceMemory : public MVKVulkanAPIDeviceObject {
 
 	/** Returns whether the memory is accessible from the host. */
     inline bool isMemoryHostAccessible() {
-#if MVK_IOS
+#if MVK_IOS_OR_TVOS || MVK_MACOS_APPLE_SILICON
```

Comment:
Hmmm...I guess at some point we might need something like?

    #define MVK_APPLE_SILICON    (MVK_IOS || MVK_TVOS || MVK_MACOS_APPLE_SILICON)